src/Symfony/Component/Routing/Tests/Fixtures/dumper/url_matcher2.php

```diff
@@ -35,7 +37,7 @@ public function match($pathinfo)
         if (0 === strpos($pathinfo, '/bar')) {
             // bar
             if (preg_match('#^/bar/(?P<foo>[^/]++)$#s', $pathinfo, $matches)) {
-                if (!in_array($this->context->getMethod(), array('GET', 'HEAD'))) {
+                if (!in_array($requestMethod, array('GET', 'HEAD'))) {
```

Comment:
I begin to wonder though, the code generating this, and executing it, become less easy to understand. Since the boost is so enormously tiny of that last bit I suggest to consider reverting that one commit and keep the rest.